drivers/ieee802154/Kconfig.cc13xx_cc26xx

```diff
@@ -91,4 +91,19 @@ config IEEE802154_CC13XX_CC26XX_SUB_GHZ_RADIO_TX_RETRIES
 	  Number of transmission attempts radio driver should do, before
 	  replying it could not send the packet.
 
+config IEEE802154_CC13XX_CC26XX_SUB_GHZ_CUSTOM_RADIO_SETUP
+	bool "Use custom radio setup structures (advanced)"
+	default n
+	help
+	  Imports the radio setup structure from an external module instead of using
+	  the default one.
+	  Allows overriding the RF modulation / protocol settings.
+
+config IEEE802154_CC13XX_CC26XX_SUB_GHZ_CUSTOM_POWER_TABLE
+	bool "Use custom power table structure (advanced)"
+	default n
+	help
+	  Imports the radio power table from an external module.
```

Comment:
We should really put this in Device Tree

Reply:
My use case here was for an application to configure custom / proprietary RF parameters (e.g., different bit rates / modulations for better signal resiliency, as generated by SmartRF Studio).
However, a Device Tree Binding for this needs to model all of the RF properties (and there are dozens). But I guess it would still be more elegant than our current approach.